### DISCUSSION TITLE
说说获取失败

### DISCUSSION THREAD

**Discussion by LioDon** [ShunCai/QZoneExport#60] [General]
由于照片数据量庞大，所以这次的备份没选取相册。扫描了两个小时后进入到下载环节。迅雷下载了前边的数据之后，打包，之后将打包的数据解压和前边的数据合并，就发现说说相关页面打不开。拓展程序页面显示插件报错，Message文件夹里只有说说相关的照片文件，没有HTML。

**Comment by LioDon** [comment]
![QQ截图20210208172932](https://user-images.githubusercontent.com/52228201/107202164-d6d41900-6a34-11eb-83ee-b6720fc8954c.png)
![QQ截图20210208172951](https://user-images.githubusercontent.com/52228201/107202175-d89ddc80-6a34-11eb-8e77-bc153ee55a56.png)
![QQ截图20210208173105](https://user-images.githubusercontent.com/52228201/107202179-d89ddc80-6a34-11eb-87c8-492fc4977f2a.png)
![QQ截图20210208173552](https://user-images.githubusercontent.com/52228201/107202182-d9367300-6a34-11eb-82a1-6da5949f3356.png)

**Reply by ShunCai** [reply]
错误那里点开堆叠追踪，或者卸载重装再试试。

**Comment by LioDon** [comment]
------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "ShunCai/QZoneExport"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2021年2月8日(星期一) 晚上6:25
收件人:&nbsp;"ShunCai/QZoneExport"<QZoneExport@noreply.github.com&gt;;
抄送:&nbsp;"LioDon"<yxc44944@hotmail.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [ShunCai/QZoneExport] 说说获取失败 (#60)





 
错误那里点开堆叠追踪，或者卸载重装再试试。

**Reply by LioDon** [reply]
重新下载压缩包，解压安装后问题依旧……

**Comment by ShunCai** [comment]
你有加我QQ吗，头像有点眼熟。

**Comment by LioDon** [comment]
没有吧，之所以眼熟是因为去年你来过我空间。我Q：1065739367，你加我吧。




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "芷炫"<notifications@github.com&gt;; 
发送时间: 2021年2月9日(星期二) 中午11:27
收件人: "ShunCai/QZoneExport"<QZoneExport@noreply.github.com&gt;; 
抄送: "yxc44944"<yxc44944@hotmail.com&gt;; "Author"<author@noreply.github.com&gt;; 
主题: Re: [ShunCai/QZoneExport] 说说获取失败 (#60)





 
你有加我QQ吗，头像有点眼熟。
